FirebaseInstallations/Source/Tests/Integration/FIRInstallationsIntegrationTests.m

```diff
@@ -140,4 +170,27 @@ - (FIRInstallationsAuthTokenResult *)getAuthToken {
   return retreivedTokenResult;
 }
 
+- (FIRInstallations *)assertInstallationsWithAppNamed:(NSString *)appName {
+  FIRApp *app = [self createAndConfigureAppWithName:appName];
+  FIRInstallations *installations = [FIRInstallations installationsWithApp:app];
+
+  XCTAssertNotNil(installations);
+  XCTAssertEqualObjects(installations.appOptions.googleAppID, app.options.googleAppID);
+  XCTAssertEqualObjects(installations.appName, app.name);
+
+  return installations;
+}
+
+#pragma mark - Helpers
+
+- (FIRApp *)createAndConfigureAppWithName:(NSString *)name {
+  FIROptions *options = [[FIROptions alloc] initInternalWithOptionsDictionary:@{
```

Comment:
Is there a reason you're using a dictionary instead of the regular init, `initWithGoogleAppID:GCMSenderID:`?

Reply:
Not really, I just didn't try `initWithGoogleAppID:GCMSenderID:`, let me try it.